tests/sentry/incidents/endpoints/test_organization_alert_rule_index.py

```diff
@@ -103,6 +103,25 @@ class AlertRuleIndexBase(AlertRuleBase):
 
     endpoint = "sentry-api-0-organization-alert-rules"
 
+    def setUp(self):
+        super().setUp()
```

Comment:
This is just a little refactoring as I realized we were using the same data in multiple tests